src/components/NavBar.js

```diff
@@ -17,18 +19,29 @@ function NavBar() {
         </Link>
 
       <div>
```

Comment:
Je zou de <div> ook binnen het {!isAuth ? : } stukje kunnen zetten; dan wordt hij pas gerenderd wannneer er geauthoriseerd is.